spec/registry.md

```diff
@@ -38,11 +38,23 @@ by any version of this specification
 for **standard** and **optional** functions.
 Such _options_ MUST use an implementation-specific _namespace_.
 
-Implementations MAY _accept_ additional _option_ values for _options_ defined here.
+Implementations MAY _accept_ additional literal _option_ values for _options_ defined here.
```

Comment:
Using "literal" here is confusing, to the extent that I'm not sure what this sentence now means. The confusion stems from our definition of the word as a part of our syntax: https://github.com/unicode-org/message-format-wg/blob/f3344b5cea59e46aa8a0aa150904f53eb7690545/spec/syntax.md?plain=1#L706-L707

An implementation (or, more accurately, a _function handler_) really should not care at all whether a value is set by a _literal_, a _variable_, or anything else.

If the intent is to refer to some other meaning of "literal", then a different word should be used.

Reply:
I do actually mean our definition of literal here. Of necessity, all values defined in the registry are string literals, either enumerated values or a reference to some serialization (like digit options or the various Unicode XXX Identifiers). The only way to have a non-literal type is by reference through a variable reference (which the second part of this PR addresses).

Would it be enough to linkify the term `literal`? Or do I need to expand on this, perhaps with an example?

Reply:
I don't think we actually have a necessity of only defining _literal_ values here.

I find it much more useful to think of these restrictions and requirements as being placed on _function handlers_, which are processing the resolved values of operands and options.

With this viewpoint, the values that a function handler processes are resolved values that may contain a character sequence value, and we're requiring that certain character sequences are accepted for certain options. This would also match our spec text on literals:
https://github.com/unicode-org/message-format-wg/blob/f3344b5cea59e46aa8a0aa150904f53eb7690545/spec/formatting.md?plain=1#L222-L224

I find this distinction rather important, even if nitpicky. It also avoids introducing confusion about the _literals_ `short` and `|short|` needing to be treated equally, as they resolve to the same value.

If "character sequence" is a bit too verbose, we could define _string_ as a term.

Reply:
So I recast the requirements here, eliminating the "literal" part altogether. I think this actually does a better job of it now. See what you think.